src/run.jl

```diff
@@ -63,12 +61,12 @@ function transform_converged(arnoldi, active, new_active, min′, V_prealloc)
     H_locked = view(arnoldi.H, active : new_active - 1, active : new_active - 1)
     schur_form = schurfact(H_locked)
 
-    V_locked = view(V_prealloc, :, active : new_active - 1)
     V_locked = view(arnoldi.V, :, active : new_active - 1)
+    V_prealloc[:, active : new_active - 1] .= V_locked
```

Comment:
I think you want to save one copy here by using V_prealloc as a temporary for the MV-product; in pseudo code

```
V_prealloc = V * Q
V = V_prealloc
```